book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def many?(rental)
+    @rentals.push(rental)
+    rental.belongs_to = self
+  end
```

Comment:
- [ ] Please follow the steps below 👇 to create the correct method

```suggestion
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

- **def add_rental(person, date):** 
  - This method allows you to add a rental for the `book`. It takes two arguments: `person` (the person renting the book) and `date` (the rental date).

- **Rental.new(date, self, person):** 
  - Inside the add_rental method, a new instance of the Rental class is created using the `Rental.new` constructor. This line establishes the association between the `book`, the `person`, and the rental `date`. The self represents the current book instance (the one on which the add_rental method is called).